flink-1.17-connector-bigquery/pom.xml

```diff
@@ -246,6 +246,10 @@ under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>false</reuseForks>
+                    <forkCount>1</forkCount>
+                </configuration>
```

Comment:
No longer needed since exo sink PR is merged

Reply:
yep, rebased.